### DISCUSSION TITLE
Release tag for Zig build system fix?

### DISCUSSION THREAD

**Discussion by dontlaugh** [jedisct1/libsodium#1416] [General]
This commit is needed to build on zig v0.14 and up (the zig master branch)

https://github.com/jedisct1/libsodium/commit/8c54312a62dd6383d692e56a65c57a2210d73936

But it's not under a tag.

The Zig build system can fetch dependencies automatically by git sha (see here for an example https://github.com/allyourcodebase/nats.c/pull/1/files). But a tag is always nicer.

Any chance we can get a tag for this fix?

**Comment by jedisct1** [comment]
Fixing compatibility issues with new compiler versions is one of the purpose of the `stable` branch.

So, I'd recommend picking a git commit from that branch (and using that branch is general).
